test/fixtures/gateway_account_fixtures.js

```diff
@@ -45,6 +45,22 @@ function validGatewayAccount (opts) {
 }
 
 module.exports = {
+  validGatewayAccountWalletToggleRequest: (opts = {}) => {
```

Comment:
Since this is in `gateway_account_fixtures` and all our patch requests require a path and value (I think `op` is always `replace` then this could be called `validGatewayAccountPatchRequest` since it need not be specific to patching wallet payments (if you remove the `|| false`).